### PR TITLE
Update Protostar to 0.11.0

### PR DESCRIPTION
**Pull request by maciektr**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -49,4 +49,4 @@ jobs:
           
       - name: Test the code
         run: | 
-          protostar test-cairo1
\ No newline at end of file
+          scarb run test
```
Comment (cptartur) [inline .github/workflows/test.yaml:52]: Should we add a comment here explaining that it's using a script from Scarb.toml?

**Review by cptartur — APPROVED** [review]

File: .github/workflows/test.yaml (hunk 1 above)
Comment (maciektr) [reply .github/workflows/test.yaml:52]: Maybe I will add a line here? https://github.com/maciektr/cairo-template/blob/fb3bf9980d1b605be4781929ba9a2528da549a57/README.md#test

Comment (cptartur) [reply .github/workflows/test.yaml:52]: Technically, there's a commend about that in Scarb.toml already. But I thought that maybe checking out `tests.yaml`, it's not immediately obvious where does `scarb run test` come from.

Comment (maciektr) [reply .github/workflows/test.yaml:52]: Hmm, the `scarb run` prefix should point to Scarb scripts anyway, so imho one place emphasising this should be enough.

Comment (cptartur) [reply .github/workflows/test.yaml:52]: Sounds reasonable, let's just keep it as it is.

**Pull request closed without merging**